### PR TITLE
Alakdan & assembly

### LINKED ISSUE #13: Switch to T-Motor or an alternative supplier instead of SunnySky
We will no longer be working with SunnySky due to quality issues and the lack of adequate customer support.

### LINKED ISSUE #15: Replace the thermistor spec
Sadulli currently uses KTY81/120 (the other models specified in the mfg docs are not used), identified as `KTY81_120` in Telega v0. As it has reached EOL, it has to be replaced with a drop-in compatible alternative that is still available for purchase, such as:

Manufacturer | Model | Note
---|---|---
NXP | KTY81/110 | 
NXP | KTY81/120 | The currently listed model
NXP | KTY81/121 | 
NXP | KTY81/122 | 
NXP | KTY81/150 | 
Littelfuse | 102PS1J |
Littelfuse | 102PS1G |

The spec needs to be updated to contain only the above-listed thermistors. All of the listed thermistors have similar R-T transfer functions.

[Littelfuse_Leaded_Thermistors_Glass_Encapsulated_Thermistors_102PS1J_RT_Table.xlsx](https://github.com/Zubax/Sadulli/files/11345684/Littelfuse_Leaded_Thermistors_Glass_Encapsulated_Thermistors_102PS1J_RT_Table.xlsx)

[KTY81_SER.pdf](https://github.com/Zubax/Sadulli/files/11345688/KTY81_SER.pdf)

### PR DESCRIPTION
**Pull request by Zentrus**
closes #15
closes #13 
- updated assembly guide
- replaced motor
- replaced thermistor specification
- updated Sadulli version
- updated Sadulli readme
- added Alakdan photos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +63,17 @@ The overcurrent protection is configured to trigger at
...
+### Sadulli v1.5 (April 2023)
+
+* Added Alakdan variant using Scorpion MII-4010 motor.
+* Removed Grosso and Piccino variants using SunnySky motors.
+* Updated assembly guide.
+* Added thermistor alternatives:
+
+Original  | Alternatives
+--------- | -----------------
+KTY81/120 | KTY81/110, KTY81/121, KTY81/122, or KTY81/150
```
Comment (pavel-kirienko) [inline README.md]:
```suggestion
KTY81/120 | KTY81/110, KTY81/121, KTY81/122, KTY81/150, 102PS1J, 102PS1G
```

File: assembly_instructions/readme.md [hunk 2]
```diff
@@ -1,36 +1,94 @@
...
+Housing bottom                | 1     |                                                                        | Milled from aluminum.
+Housing screw                 | 4     | DIN 965 20 mm / M2.5x16                                                | Fastening both parts of the housing together.
+Motor screw                   | 4     | DIN 7984 4 mm / M3x4                                                   | Fastening the motor to the top part of the housing.
+Thermal pad                   | Dia 42 mm | Arctic Cooling ACTPD00018A, ACTPD00005A, or ACTPD00002A            | Sinks the heat to the bottom part of the housing.
+Connectors PCB                | 1     |                                                                        |
+Logic PCB                     | 1     |                                                                        |
+Power stage PCB               | 1     |                                                                        |
+Thermistor                    | 1     | KTY81/120; Alternatives: KTY81/110, KTY81/121, KTY81/122, or KTY81/150 | PTC thermistor with pre-soldered extension wires.
```
Comment (pavel-kirienko) [inline assembly_instructions/readme.md]:
```suggestion
Thermistor                    | 1     | NXP KTY81/120, NXP KTY81/110, NXP KTY81/121, NXP KTY81/122, NXP KTY81/150, Littelfuse 102PS1J, or Littelfuse 102PS1G | PTC thermistor with pre-soldered extension wires.
```

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]
Some things in this list are not directly related to the motor replacement but are still necessary to do.

- [x] Replace UAVCAN with Cyphal and add links to telega.zubax.com
- [x] We need to increase the hardware overcurrent protection resistance to 27k (R1 on the `connectors` board). There is no purpose in providing this information in the README.
- [x] Remove all generated files from the repository. Repositories are meant to contain only source files. Everything that can be generated from the source files goes into documentation package releases.

File: assembly_instructions/readme.md [hunk 3]
```diff
@@ -1,36 +1,94 @@
...
+    * Refer to the images below for the proper placement.
 
-**The phase wires shall be soldered such that a positive setpoint drives the motor clockwise (viewed from the top).**
+<p align="center">
+<img src="figures/1-logic.jpg" alt="" width="500"/><img src="figures/2-logic-top.jpg" alt="" width="500"/><br>
+<img src="figures/3-logic-bottom.jpg" alt="" width="500"/><img src="figures/4-mitochondrik-logic.jpg" alt="" width="500"/>
+</p>
+
+3. Motor modification. 
+    * Remove the motor stopper.
```
Comment (pavel-kirienko) [inline assembly_instructions/readme.md]:
```suggestion
    * Remove the motor retainer ring.
```

File: assembly_instructions/readme.md [hunk 4]
```diff
@@ -1,36 +1,94 @@
...
+
+<p align="center">
+<img src="figures/7-thermistor.jpg" alt="" width="500"/><img src="figures/8-epoxy.jpg" alt="" width="500"/><br>
+<img src="figures/9-coil.jpg" alt="" width="500"/><img src="figures/10-trim.jpg" alt="" width="500"/><br>
+<img src="figures/11-wires.jpg" alt="" width="500"/><img src="figures/12-insert.jpg" alt="" width="500"/><br>
+</p>
+
+5. Housing Attachment. See photos for reference.
+    * Insert the motor phase wires and the thermistor wires through the housing holes. Ensure proper placement.
+    * Get 4 pieces of DIN 7984 4 mm (motor screws). Attach the motor to the top housing. Use threadlock.
```
Comment (pavel-kirienko) [inline assembly_instructions/readme.md]:
```suggestion
    * Get 4 pieces of DIN 7984 4 mm (motor screws). Attach the motor to the top housing. Use threadlock glue.
```

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]
I left two minor suggestions. Aide from that, the only remaining issue is to update the resistor I mentioned earlier.

**Comment by Zentrus** [comment]
@abualkasim-bakeer can you increase the hardware overcurrent protection resistance to 27k (R1 on the connectors board) in eagle?

**Comment by abualkasim-bakeer** [comment]
> @abualkasim-bakeer can you increase the hardware overcurrent protection resistance to 27k (R1 on the connectors board) in eagle?

Okay, I will check it after finishing Valenok. Or you need it urgent?

**Comment by abualkasim-bakeer** [comment]
The required change is done.

**Comment by pavel-kirienko** [comment]
@abualkasim-bakeer please open a pull request against this branch.

image

**Review by pavel-kirienko — APPROVED** [review]

**Pull request merged**